src/lib/auth/README.md

```diff
@@ -0,0 +1,45 @@
+## Authentication Library
+Initialize authentication library (authenticator). Based on the authentication type a strategy
```

Comment:
> Based on the authentication type a strategy will be prepared.

This sentence is a bit faulty IMO. I would either rewrite as *"A strategy will be prepared based on the authentication type"*, or add a comma to distinguish the clauses e.g. *"Based on the authentication type, a strategy will be prepared"*.

Reply:
Also happening on line 18.

Reply:
Definitely more clear, updated, thanks.